content/hardware/09.kits/maker/nesso-n1/datasheet/datasheet.md

```diff
@@ -350,6 +354,19 @@ Use the limits below to define the operating environment, thermal margins, and p
 
 <div style="page-break-after: always;"></div>
 
+### Programming Options
+
+Nesso N1 supports multiple programming methods:
+
+- **Arduino IDE** [1]
+- **Arduino Cloud Editor** [2]
```

Comment:
```suggestion
```

Although it is possible to use the Nesso N1 as an Arduino Cloud IoT Device, programmed via one of the other methods listed here ([example](https://forum.arduino.cc/t/nesso-n1-not-showing-on-arduino-cloud-editor-despite-the-manual-saying-so/1414912/8)). Arduino Cloud Editor does not currently have support for the board (see PXCT-1579 in the internal tracker).